docs_src/use-cases/use-cases.md

```diff
@@ -1,4 +1,5 @@
 # Intel Retail Use Cases
 
 [Automated Self Checkout](./automated-self-checkout/automated-self-checkout.md)
-[AI Connect for Scientific Data (AiCSD)](./AiCSD/aicsd.md)
\ No newline at end of file
+[AI Connect for Scientific Data (AiCSD)](./AiCSD/aicsd.md)
+[Retail Use Cases: C-API for YOLOV8 ensemble](./capi-yolov8-ensemble/capi-yolov8-ensemble.md)
```

Comment:
can we make this as bullet points ? Currently all of them are displayed on one single line

Reply:
fixed